tests/UI/specs/PivotByDimension_spec.js

```diff
@@ -20,10 +20,8 @@ describe("PivotByDimension", function () {
         await element.click();
         await page.waitForNetworkIdle();
 
-        await page.evaluate(function(){
-            $('.dropdownConfigureIcon').click();
-            $('.dataTablePivotBySubtable').click();
-        });
+        await page.click('.dropdownConfigureIcon');
+        await page.click('.dataTablePivotBySubtable');
```

Comment:
Should we add a small delay between the clicks? I recall seeing that in other tests where we do use the dropdown it's done similarly (haven't looked for exact places though).

```suggestion
        await page.click('.dropdownConfigureIcon');
        await page.waitForTimeout(100);
        await page.click('.dataTablePivotBySubtable');
```

Reply:
It should at least not make more problems.